src/lib/components/_all.scss

```diff
@@ -1,3 +1,4 @@
 @import './ProfileSwitcher/styles';
 @import './ProgressBar/styles';
-@import './SparklineChart/styles';
\ No newline at end of file
+@import './SparklineChart/styles';
+@import './UrlChooser/styles';
```

Comment:
newline

Reply:
did the linter not grump about this? maybe I broke it at some point... I think it used to block on this.

Reply:
Fixed